src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -64,7 +150,15 @@ def _refine_log_line(self, line):
         elif (
             self.workflow_or_task_run_id is None
         ) or self.workflow_or_task_run_id in line:
-            return line
+            if line.startswith("{"):
```

Comment:
**thought**: the `json.loads()` approach above would also tell us if a line is a JSON or plain text. This is a situation where EAFP might be better than LBYL. [Ext resource on the topic](https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/).